pkgs/tools/filesystems/dbxfs/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib, fetchgit, python3Packages, fusePackages }:
+
+let
+  block-tracing = python3Packages.buildPythonPackage rec {
+    pname = "block_tracing";
+    version = "1.0.1";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
+      sha256 = "9faa009a702a8f2f605ebb78314d5ca2a2a93543d061038a3d3a978c93385e68";
+    };
+  };
+  privy = python3Packages.buildPythonPackage rec {
+    pname = "privy";
+    version = "6.0.0";
+    format = "wheel";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
```

Comment:
```suggestion
      inherit pname version;
```

Reply:
applied in https://github.com/NixOS/nixpkgs/pull/155134/commits/6b30949169f06715eb8973b6ddb13b19467be98a